### ISSUE TITLE
Connect multiple Go Pros with multiple wifi adapters.

### ISSUE THREAD

**Issue by josebasgmz** [KonradIT/goprowifihack#198]
**Problem:**

I'm interested in streaming from two Go Pros at the same time, but as far as i know this is not possible using bluetooth. I was thinking about connecting the Ad Hoc wifi networks created by the cameras with two different wifi adapters.  Is it possible?

Thanks!
 **Details:**

GoPro Camera(s): 2 Go Pro hero 7 Black

**Comment by demiantres** [comment]
I do not think this will work. But you can start both cameras in the Live Streaming mode using the GoPro app.
